### ISSUE TITLE
Auths of everything by owner

### ISSUE THREAD

**Issue by sundy-li** [line/promgen#217]
I just create a pr #210  and fixed the ServiceList filter codes.

But I found that the   ProjectList, RulesList (/rule),  UrlsList, NotifiorsList will display all the entries by default.
It's not very safe because we have lots of users to manage different projects. We should ensure that each user could only have the update privilege of there own projects,rules, urls etc.

**Comment by kfdm** [comment]
Right, currently the recommended way to handle this, is a promgen instance per-team. In the future I would like to better handle groups but I'm trying to refactor some other parts of Promgen before I try to tackle groups

**Comment by sundy-li** [comment]
Ok.

And I'd like to create some PR to improve promgen, but I found that there are many conflicts of code, it's hard to track different branches.

So I will create some issues later which may be fixed in my version.

**Comment by vincent-olivert-riera** [comment]
A new permission system has been introduced in [v0.73.0](https://github.com/line/promgen/releases/tag/v0.73.0).
